libs/vscode/tasks/src/lib/cli-task-commands.ts

```diff
@@ -49,6 +49,10 @@ export function registerCliTaskCommands(
     );
   });
 
```

Comment:
I am not really sure of the distinction between nx-task-commands and cli-task-commands. It feels like this cli-task-commands was originally angular CLI support vs Nx CLI. I started out putting the run command support in nx-task-commands, but the flow fit in a little better here with the other CLI commands and it has all the support for getting the executor/builder definition and prompting for flags.

Reply:
yea, I noticed that as well. We'll eventually consolidate the two and just use Nx.